zebrad/tests/acceptance.rs

```diff
@@ -256,22 +255,27 @@ fn serialized_tests() -> Result<()> {
 fn valid_generated_config() -> Result<()> {
     zebra_test::init();
 
-    // Generate configuration in binary parent path
-    let binary_path = Path::new(env!("CARGO_BIN_EXE_zebrad")).parent().unwrap();
-    let mut path = PathBuf::from(binary_path);
-    path.push("zebrad.toml");
-    let (child, _guard) = get_child(&["generate", "-o", path.to_str().unwrap()])?;
+    // Get a temp dir
+    let tempdir = ZebraTestDir::new("zebrad_tests");
+    let mut path_config = PathBuf::from(tempdir.path());
+    path_config.push("zebrad.toml");
+
+    // Generate configuration in temp dir path
+    let tempdir_string = path_config.parent().unwrap().to_str().unwrap();
+    let config_string = path_config.to_str().unwrap();
+    let (child, tempdir) = get_child(&["generate", "-o", config_string], None)?;
+
     let output = child.wait_with_output()?;
     output.assert_success()?;
 
     // Check if the file was created
-    assert_eq!(path.exists(), true);
+    assert_eq!(path_config.exists(), true);
 
-    // Check if it was created in the same path as binary
-    assert_eq!(path.parent(), Some(binary_path));
+    // Check if it was created in the tempdir
+    assert_eq!(PathBuf::from(path_config.parent().unwrap()), tempdir.path());
```

Comment:
rather than unwrapping I'd use a pattern for the match
```suggestion
    assert_eq!(path_config.parent(), Some(tempdir.path()));
```